### DISCUSSION TITLE
Bug: PR title contains a version if multiple, independent dependencies are upgraded to the same version

### DISCUSSION THREAD

**Discussion by fdw** [renovatebot/renovate#28845] [Request Help] [answered]
### What would you like help with?

I think I found a bug

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

BitBucket, latest version

### Please tell us more about your question or problem

I group non-major dependencies into one large PR. By chance, in one run, there were only two available updates, both to the same version. The title of the PR read "Update non-major to 1.2.0", which is unexpected ;)

**Comment by rarkins** [comment, marked as answer]
It's by design, as it's usually desirable

**Reply by fdw** [reply]
Okay, thank you 🙂
